piff/star.py

```diff
@@ -941,22 +961,72 @@ def alpha(self):
     def beta(self):
         return self.A.T.dot(self.b)
 
-    def newParams(self, params, **kwargs):
+    @classmethod
+    def reshape_params(cls, params, params_lens):
+        new_params = np.empty(len(params_lens), dtype=object)
```

Comment:
`dtype=object` always feels awkward to me.  Is there a reason params can't just be a list or list of lists?

Reply:
Yeah, that's probably a better way to do this.  Not sure why I got myself stuck on it being a numpy.array.